internal/jstest/jstest.go

```diff
@@ -53,27 +55,52 @@ func Run(ctx context.Context, dir string, args []string) (*internal.TestResults,
 		TestResults: make(map[string]internal.TestResult),
 	}
 
+	type item struct {
+		file string
+		err  error
+		out  []byte
+	}
+
 	volume := "tests"
-	for _, testName := range files {
-		output, err := runCommand(dir, "mongo", filepath.Join(volume, testName))
-		if err != nil {
-			if _, ok := err.(*exec.ExitError); !ok {
-				return nil, err
+
+	ch := make(chan *item, len(files))
+
+	var wg sync.WaitGroup
```

Comment:
We run all files in parallel there, without an upper bound. That's fine now, but something we could improve in the future